dotnet/src/SemanticKernel/Planning/FunctionFlowRunner.cs

```diff
@@ -259,8 +259,10 @@ private async Task<string> ProcessNodeListAsync(XmlNodeList nodeList, string fun
 
                     if (processFunctions && !string.IsNullOrEmpty(functionName))
                     {
-                        Verify.NotNull(functionName, nameof(functionName));
-                        Verify.NotNull(skillFunction, nameof(skillFunction));
+                        if (skillFunction is null)
```

Comment:
Someone will need to resolve conflict with https://github.com/microsoft/semantic-kernel/pull/523 -- making note for self.

Reply:
I beat you :)

See `Plan.cs`, `SequentialPlanParser.cs`, and `SequentialPlanner.cs` for applying changes.